<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>바인딩쿼리변환</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>바인딩 쿼리 입력</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>바인딩 파라미터 입력</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>리터럴 쿼리 결과</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_BindingQuery">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_BindingParameter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_literalQuery">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
